### ISSUE TITLE
Workflow, tool version outline

### ISSUE THREAD

**Issue by s-junguy** [jtamames/SqueezeMeta#825]
Hi, is there a way I can get access to a list of tools and their version I ran within the pipeline?

**Comment by s-junguy** [comment]
Thanks for your help but I not only need the version from the pipeline but also the versions from every single tool that has been executed within the pipeline. If I click on the links under "18. License and third-party software", I only get to the github site.

**Comment by jtamames** [comment]
The versions of the tools are the same than the pipeline

**Comment by s-junguy** [comment]
If I execute ```SqueezeMeta.pl -v``` I only get 
```
***
1.6.3, September 2023
```
but I also need the versions from every tool like trimmomatic, megahit etc. I still don't know where to find these versions...

**Comment by jtamames** [comment]
The tool list and URLs leading to tool's repos can be found in https://github.com/jtamames/SqueezeMeta#18-license-and-third-party-software

**Comment by fpusan** [comment]
Yeah but this doesn't include the actual versions that we are calling within SqueezeMeta. In most cases it is the latest version, but that can not be taken for granted
